### PR TITLE
Kokoro continuous build setup and windows build fix

### LINKED ISSUE #475: MacOS and Windows Continuous builds
We recently committed a bug that only appeared on Macs. 

Probably use kokoro.

### LINKED ISSUE #480: Fix Windows build
For #475 . As of now the test fails with Windows continuous build. Let's fix it.

# Fix: checking suffix of a path (toString)

```
[ERROR] Failures: 
[ERROR]   LinkageCheckerTest.testFindClassReferences_privateClass:619 When the superclass is unavailable, it should report the location of InnerService
expected to end with: testdata/api-common-1.7.0.jar
but was             : T:\src\github\cloud-opensource-java\dependencies\target\test-classes\testdata\api-common-1.7.0.jar
```

# Illegal character ":" for Path

```
java.nio.file.InvalidPathException: Illegal char <:> at index 2: /T:/src/github/cloud-opensource-java/dashboard/target/classes/css/dashboard.css
	at sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:182)
	at sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:153)
	at sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
	at sun.nio.fs.WindowsPath.parse(WindowsPath.java:94)
	at sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:255)
	at java.nio.file.Paths.get(Paths.java:84)
	at com.google.cloud.tools.opensource.dashboard.DashboardMain.copyResource(DashboardMain.java:146)
	at com.google.cloud.tools.opensource.dashboard.DashboardMain.generateHtml(DashboardMain.java:126)
	at com.google.cloud.tools.opensource.dashboard.DashboardMain.generate(DashboardMain.java:114)
	at com.google.cloud.tools.opensource.dashboard.DashboardTest.setUp(DashboardTest.java:84)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
[ERROR] Tests run: 2, Failures: 1, Errors: 1, Skipped: 0, Time elapsed: 107.281 s <<< FAILURE! - in com.google.cloud.tools.opensource.dashboard.DashboardTest
[ERROR] com.google.cloud.tools.opensource.dashboard.DashboardTest  Time elapsed: 107.281 s  <<< FAILURE!
java.lang.AssertionError: Could not generate dashboard
	at com.google.cloud.tools.opensource.dashboard.DashboardTest.setUp(DashboardTest.java:87)
```

### PR DESCRIPTION
**Pull request by suztomo**
Fixes #475.

This repository already has existing job for [period jobs for Ubuntu](https://github.com/GoogleCloudPlatform/cloud-opensource-java/tree/master/kokoro/ubuntu). This change is setup for continuous jobs for Mac, Windows, and Ubuntu.

I followed [appengien-plugins-core's Kokoro configuration](https://github.com/GoogleCloudPlatform/appengine-plugins-core/blob/master/kokoro/continuous.bat), and tested with Kokoro QA (+ Fusion QA), example job "codelab/suztomo/macos_external/continuous".

Windows build is fixed. Fixes #481.

After this PR is merged:
- Setup prod Kokoro job (cl/236179915)
- add `kokoro-team` to repository admin (done)
- Register kokoro-github-subscriber

With Kokoro QA, it worked:

image

### CODE REVIEW

File: kokoro/continuous.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Fail on any error.
+set -e
+# Display commands being run.
+set -x
+
+export M2_HOME=/usr/local/apache-maven
```
Comment (chanseokoh) [inline kokoro/continuous.sh]: I've read here and there that you don't want to use `M2_HOME` going forward.

[Maven 3.5.0 release note](https://maven.apache.org/docs/3.5.0/release-notes.html)
https://stackoverflow.com/a/47712646/1701388
https://stackoverflow.com/a/17137183/1701388

Comment (suztomo) [reply kokoro/continuous.sh]: Removed M2_HOME.

**Comment by suztomo** [comment]
Now it passes Windows build.

```
[INFO] Google Cloud Platform Supported Libraries .......... SUCCESS [  3.062 s]
[INFO] BOM Parent ......................................... SUCCESS [  0.016 s]
[INFO] Cloud Platform Supported Libraries Upper Bounds Check SUCCESS [  8.797 s]
[INFO] Cloud Tools Open Source Code Hygiene Tooling ....... SUCCESS [  0.015 s]
[INFO] Dependency Lister .................................. SUCCESS [02:55 min]
[INFO] Cloud Tools Open Source Code Hygiene Dashboard ..... SUCCESS [02:08 min]
[INFO] Linkage Checker Enforcer Rule ...................... SUCCESS [ 28.583 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 05:46 min
[INFO] Finished at: 2019-02-28T08:13:16-08:00
[INFO] Final Memory: 32M/318M
[INFO] ------------------------------------------------------------------------

T:\src\github\cloud-opensource-java>exit /b 0 


[ID: 9541559] Build finished after 351 secs, exit value: 0
```

File: dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java [hunk 2]
```diff
@@ -141,9 +144,10 @@ private static Path generateHtml(ArtifactCache cache,
     return output;
   }
 
-  private static void copyResource(Path output, String resourceName) throws IOException {
+  private static void copyResource(Path output, String resourceName)
+      throws IOException, URISyntaxException {
     ClassLoader classLoader = DashboardMain.class.getClassLoader();
-    Path input = Paths.get(classLoader.getResource(resourceName).getPath());
```
Comment (suztomo) [inline dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java]: This wasn't working for Windows. Error noted in #475

File: dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerTest.java [hunk 3]
```diff
@@ -615,8 +615,8 @@ public void testFindClassReferences_privateClass() throws IOException, URISyntax
     Truth.assertThat(classReferenceError.getReason()).isEqualTo(Reason.INACCESSIBLE_CLASS);
     Truth.assertWithMessage(
             "When the superclass is unavailable, it should report the location of InnerService")
-        .that(classReferenceError.getTargetClassLocation().toString())
-        .endsWith("testdata/api-common-1.7.0.jar");
```
Comment (suztomo) [inline dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerTest.java]: This wasn't working for Windows. Error noted in #475

**Pull request merged**